packages/manager/modules/pci/src/projects/project/storages/databases/databases.constants.js

```diff
@@ -1,15 +1,47 @@
 import get from 'lodash/get';
 import head from 'lodash/head';
+import ASSET_MONGO_DB from './assets/mongodb.png';
+import ASSET_KAFKA from './assets/kafka.png';
+import ASSET_MYSQL from './assets/mysql.png';
+import ASSET_POSTGRE_SQL from './assets/postgresql.png';
+import ASSET_REDIS from './assets/redis.png';
+
+export const ENGINE_LOGOS = {
+  mongodb: ASSET_MONGO_DB,
+  kafka: ASSET_KAFKA,
+  mysql: ASSET_MYSQL,
+  postgresql: ASSET_POSTGRE_SQL,
+  redis: ASSET_REDIS,
+};
 
 export const DATABASES_GUIDES_URL =
   'https://docs.ovh.com/gb/en/publiccloud/databases/';
 
 export const DATABASE_TYPES = {
   MONGO_DB: 'mongodb',
+  MYSQL: 'mysql',
+  POSTGRESQL: 'postgresql',
+  REDIS: 'redis',
```

Comment:
This should be used in other places instead of hard-coding it everywhere

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1